CIP-_/README.md

```diff
@@ -0,0 +1,209 @@
+---
+CIP: "?"
```

Comment:
```suggestion
CIP: ?
```
A basic formatting requirement as per CIP-0001.

Reply:
Thank you! I will update the formatting tomorrow =)

Reply:
sounds great... I'll commit the essentials now & please let's do the commits gradually rather than force pushing from your end, since one or two of these issues might benefit from group discussion going forward & being able to track the changes. 😎